src/motion/__tests__/component.test.tsx

```diff
@@ -243,27 +243,6 @@ describe("motion component rendering and styles", () => {
         expect(container.firstChild).toHaveStyle("background-color: #f00")
     })
 
-    // Skipping this test because originX and originY aren't correctly detected by
-    // @testing-library/react, it returns a false positive.
-    it.skip("style supports originX and originY", () => {
-        const { container } = render(<motion.div style={{ originX: "25%" }} />)
-        expect(container.firstChild).toHaveStyle("transform-origin: 25% 50%")
-    })
-
-    it("updates style for transform values", () => {
-        let rotate = 0
-
-        const Component = () => {
-            rotate += 45
```

Comment:
This was an unsafe mutation and this use-case is covered in the build-html-styles tests